### PR TITLE
Feature load picture

### PR DESCRIPTION
**Pull request by Mikhail-Kh**

### CODE REVIEW

File: app/src/main/java/by/khryshchanovich/androidapplication/MainActivity.kt [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
+import android.content.Intent  // подключаем класс Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import kotlinx.android.synthetic.main.activity_main.*
 
 class MainActivity : AppCompatActivity() {
+
+    override fun onStart() {
+        super.onStart()
+    }
```
Comment (andrewKarachun0304) [inline app/src/main/java/by/khryshchanovich/androidapplication/MainActivity.kt]: I think this is unnecessary

**Review by andrewKarachun0304 — APPROVED** [review]
Cool!

File: app/src/main/java/by/khryshchanovich/androidapplication/MainActivity.kt (hunk 1 above)
Comment (TMSlebedev) [reply app/src/main/java/by/khryshchanovich/androidapplication/MainActivity.kt]: Remove

**Review by TMSlebedev — APPROVED** [review]

**Comment by Mikhail-Kh** [comment]
I removed the unnecessary method

**Pull request merged**